types/node-targz/package.json

```diff
@@ -11,10 +11,5 @@
     "devDependencies": {
         "@types/node-targz": "workspace:."
     },
-    "owners": [
-        {
-            "name": "Mats Jun Larsen",
-            "githubUsername": "matsjla"
```

Comment:
@junlarsen Do you still want to be the owner of this package?

Reply:
Hi yes, that'd be great thank you